src/platforms/apple/common/install/swift-package-manager.mdx

```diff
@@ -14,7 +14,7 @@ You can define your dependency rule by selecting the SDK version (or branch), an
 Alternatively, when using SPM version 5.2 or later and your project uses `Package.swift` file to manage dependencies, you can specify the target with name:
 
 ```swift {tabTitle:Swift}
-.package(name: "Sentry", url: "https://github.com/getsentry/sentry-cocoa", from: "{{ packages.version('sentry.cocoa') }}"),
+.package(url: "https://github.com/getsentry/sentry-cocoa", from: "{{ packages.version('sentry.cocoa') }}"),
```

Comment:
We could also change it to upToNextMajor
```suggestion
.package(url: "https://github.com/getsentry/sentry-cocoa", .upToNextMajor(from: "{{ packages.version('sentry.cocoa') }}"))
```

Reply:
It seems that the `from:` argument already has this logic, from [`.package(...)` documentation](https://github.com/apple/swift-package-manager/blob/dd7e9cc68fccd6aad5db11aca2becfc24746905a/Sources/PackageDescription/PackageDependency.swift#L230-L231):

>Adds a package dependency that uses the version requirement, starting
>with the given minimum version, going up to the next major version.

Reply:
Okay, then we can leave it.